backend/impl/build.gradle

```diff
@@ -14,6 +14,8 @@ dependencies {
   api project(':backend:api')
   api project(':branchLayout:api')
 
+  // Note that we can't easily use Gradle's `testFixtures` configuration here
+  // as it doesn't seem to expose testFixtures resources in test classpath correctly.
```

Comment:
Unrelated observation so that I don't reattempt using https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures